### DISCUSSION TITLE
How should alert language change for SRKWs in 2025?

### DISCUSSION THREAD

**Discussion by scottveirs** [salish-sea/whale-ocean-alert#12] [General]
On Jan 1, 2025, the law in Washington State will change regarding approach distances for SRKWs. The details are provided in Revised Code of Washington: [RCW 77.15.740](https://app.leg.wa.gov/RCW/default.aspx?cite=77.15.740) (see also screenshot below).

Let's discuss:

**What is the optimal update of guidance for Washington-based recipients of alerts?**

<img width="1601" alt="Screenshot 2024-09-30 at 11 48 27 AM" src="https://github.com/user-attachments/assets/a9c2a5c6-62f8-4bec-ac23-cd54a3474b90">

**Comment by scottveirs** [comment]
See also [https://www.bewhalewise.org/killer-whales/#us-regulations](https://www.bewhalewise.org/killer-whales/#us-regulations)

**Comment by serenatierra** [comment]
These are the main points Orca Network is planning to highlight in our materials:
Maintain a minimum distance of 1000 yards from SRKWs
Slow to under 7 knots if within 1000 yards of SRKWs
Disengage the vessel transmission (if safe to do so) if SRKWs surface within 400 yards
Treat all orcas as SRKWs until IDs are confirmed

We are trying not to use the word "approach" in our updated graphics and Share the Water materials. The overall messaging we have heard from WDFW is that they want boaters to avoid SRKWs, rather than trying to approach/view them from 1000 yards.

**Comment by vzett** [comment]
This is a forthcoming update to the existing bewhalewise guidelines which
will have a phone and tablet version.  Does anyone know when a new Be Whale
Wise doc will be available?

We can update wording in the area / sighting popups to whatever we want but
we are trying to keep the guidelines tied to the most "official" source
whenever possible.

Thanks
Virgil
